Overview.src.html

```diff
@@ -2663,6 +2685,30 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <p><span title="Read a request">Read <var>request</var></span>.
 
  <li>
+  <p>Let <var>strategy</var> be an object. The user agent may choose any object.
```

Comment:
Maybe in the future we should define "valid queuing strategy". Right now this could be taken to allow any object, including ones that cause the constructor to throw. For now it seems fine as-is.